src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: var(--transition-second);
```

Comment:
Also specify properties you change on hover to improve performance

Reply:
I did not understand what exactly I should indicate.

Reply:
`transition: transform var(--transition-second), color var(--transition-second);`

See more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property)

If you have any questions - feel free to ask them in fe_chat